tele_giphy/game/models.py

```diff
@@ -13,22 +14,23 @@ def __str__(self):
         return self.token
 
 
-# Keeps track of users in relation to game tokens
-class User(models.Model):
-    username = models.CharField(max_length=60)
+# Keeps track of what game is attached to a user
+class UserGame(models.Model):
+    user = models.OneToOneField(User)
```

Comment:
No, the usernames have to be unique since I'm using the default django user model. So that checking is taken care of when it's created, see https://github.com/JessicaNgo/TeleGiphy/blob/b203cf5968c8a10849bcd009b1de77c277d4f220/tele_giphy/game/views.py#L152